tests/pipeline/test_column.py

```diff
@@ -91,23 +92,33 @@ def test_latest(self):
         (operator.le,),
     ])
     def test_comparison_errors(self, op):
-        columns = TDS.columns
-
-        def get_error_msg(op, compare_type):
-            return (
-                "'{op}' not supported between instance of "
-                "'{compare_type}' and '.*'. "
-                "Did you mean use '.latest' with '.*'?"
-            ).format(op=op, compare_type=compare_type)
-
-        ops = {'gt': '>', 'ge': '>=', 'lt': '<', 'le': '<='}
+        for column in TDS.columns:
+            with self.assertRaises(TypeError):
+                op(column, 1000)
+            with self.assertRaises(TypeError):
+                op(1000, column)
+            with self.assertRaises(TypeError):
+                op(column, 'test')
+            with self.assertRaises(TypeError):
+                op('test', column)
+
+    def test_comparison_error_message(self):
+        column = USEquityPricing.volume
+        err_msg = (
+            "'<' not supported between instance of"
+            " 'int' and 'EquityPricing<US>.volume'."
+            " Did you mean to use 'EquityPricing<US>.volume.latest'?"
+        )
 
-        for column in columns:
-            int_err_msg = get_error_msg(ops[op.__name__], 'int')
-            str_err_msg = get_error_msg(ops[op.__name__], 'str')
+        with self.assertRaises(TypeError) as e:
+            column < 1000
+            self.assertEqual(str(e), err_msg)
```

Comment:
This needs to happen outside the `with` block, otherwise it won't get hit.

Reply:
Good find, think I fixed it by unindenting and then doing `self.assertEqual(str(e.exception), err_msg)`